controllers/favorites.js

```diff
@@ -0,0 +1,90 @@
+var express = require("express");
+var router = express.Router();
+var db = require('../models');
+// current directory : localhost:3000/favorites
+// router.use(express.static(__dirname + '/static'));
+
+/// localhost:3000/favorites/
+router.get("/", function(req, res) {
+	db.favorite.findAll().then(function(fav) {
+		res.render('favorites.ejs', {fav:fav})
+	})
+	//this is where I will list the database entries
+});
+
+router.post("/", function(req, res) {
+	var title = req.body.favtitle;
+	var year = req.body.favyear;
+	var imdb = req.body.favimdb;
+
+	db.favorite.create({
+		imdbId: imdb,
+		title: title,
+		year: year
+	}).then
+	res.redirect("/movies/" + imdb);
+})
+
+router.get("/tags", function(req, res) {
```

Comment:
We'll want to include associated favorites so that we can list them under the tag on the page.
```
router.get("/tags", function(req, res) {
	db.tag.findAll({
		include: [db.favorite]
	})
	.then(function(alltags) {
		res.render('allTags.ejs', {alltags:alltags})
	});
});
```